.codeclimate.yml

```diff
@@ -1,7 +1,5 @@
 ---
 engines:
-  golint:
-    enabled: true
```

Comment:
`golangci-lint` is not supported in CodeClimate, but I've added it to the `.travis.yml` file along with local run instructions.